pkg/trace/stats/concentrator_test.go

```diff
@@ -94,15 +113,20 @@ func TestConcentratorOldestTs(t *testing.T) {
 			t.FailNow()
 		}
 
-		// First oldest bucket aggregates old past time buckets, it should have it all.
-		for key, count := range stats[0].Counts {
-			if key == "query|duration|env:none,resource:resource1,service:A1" {
-				assert.Equal(151, int(count.Value), "Wrong value for duration")
-			}
-			if key == "query|hits|env:none,resource:resource1,service:A1" {
-				assert.Equal(6, int(count.Value), "Wrong value for hits")
-			}
+		// First oldest bucket aggregates old past time buckets, so each count
+		// should be an aggregated total across the spans.
+		expected := map[string]float64{
+			"query|duration|env:none,resource:resource1,service:A1":           151,
+			"query|hits|env:none,resource:resource1,service:A1":               6,
+			"query|errors|env:none,resource:resource1,service:A1":             0,
+			"custom_query_op|duration|env:none,resource:resource1,service:A1": 500,
+			"custom_query_op|hits|env:none,resource:resource1,service:A1":     1,
+			"custom_query_op|errors|env:none,resource:resource1,service:A1":   0,
+			"nested_op|duration|env:none,resource:resource1,service:A1":       2500,
+			"nested_op|hits|env:none,resource:resource1,service:A1":           2,
+			"nested_op|errors|env:none,resource:resource1,service:A1":         1,
```

Comment:
The expected data for the test also ensures that we do not somehow overcount across operations - be they top-level or measured.